app/src/main/java/ru/practicum/android/diploma/RootActivity.kt

```diff
@@ -0,0 +1,40 @@
+package ru.practicum.android.diploma
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.isVisible
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.setupWithNavController
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+class RootActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_root)
+
+        // Пример использования access token для HeadHunter API
+        networkRequestExample(accessToken = BuildConfig.HH_ACCESS_TOKEN)
+
+        val navHostFragment = supportFragmentManager.findFragmentById(R.id.rootFragmentContainerView) as NavHostFragment
+        val navController = navHostFragment.navController
+
+        val bottomNavigationView = findViewById<BottomNavigationView>(R.id.bottomNavigationView)
+        bottomNavigationView.setupWithNavController(navController)
+
+        navController.addOnDestinationChangedListener { _, destination, _ ->
+            when (destination.id) {
+                R.id.jobFragment -> {
+                    bottomNavigationView.isVisible = false
+                }
```

Comment:
```suggestion
                R.id.jobFragment -> bottomNavigationView.isVisible = false
```
Если лябмда однострочная, то лучше писать в 1 строку без {}